### DISCUSSION TITLE
Many Ideas

### DISCUSSION THREAD

**Discussion by qianxliu** [zoo-js/zoo#247] [Ideas] [answered]
@xrkffgg 
@ajuner  

好像在这个工程活跃的就你们两个……

- [ ] 可不可以建一个类似你主页的Quote Repos，因为有些格言觉得可以分享给所有人，而且这个确实没什么难度
- [ ]  我主要是做机器学习深度学习和Android还有游戏方面的，不知道乱添加 Repos 会不会打乱布局（https://github.com/Neteraxe/Body-Pix-Demo  ）
- [ ]  网页使用 gitment 实现评论
- [ ]  可以出个 Contributions 指南
- [ ]  （重要）<b>https://gitter.im/ </b>, we don't need WeChat or QQ
- [ ]  一些工程应添加正确的LICENSE，以保证知识版权的合理性
- [ ]  还有zoo-js这个名字……为什么只有js(笑)。技术栈太偏前端了

**Comment by qianxliu** [comment]
这些建议都是比较广泛的建议，建议
https://github.com/zoo-js/zoo/projects
还有就是虽然有很多repos，但是一个repo能留下一个可以广泛使用的项目，都是很难的。也许应该把人聚集起来。
What we do may be small, but it has a certain character of permanence.
以上都是微不足道的建议

**Reply by qianxliu** [reply]
虽然有将近几十个orgs，但是org人少其实并不具备可以出现能广泛使用的repo的条件
而且js技术栈真的很容易摸鱼……

**Comment by qianxliu** [comment]
还有就是我认为后端也很重要，而且后端多是程设方面的，有较好的代码质量
JavaScript技术栈太前端了，主要讲Hack，而不是讲Research，社区需要更多运营
运营方面的建议在我的第4条建议中给了很好的答案
我主要还是Research方面的，其实不太做代码分享和社交，最近正好有一点时间便提一点建议

**Comment by xrkffgg** [comment, marked as answer]
1. zoo-js 更名现在其实不太那么恰当吧，至于为啥 js ，其实起初并没有考虑那么多的技术栈之类的。其实我觉得这个只是个代号了，并不局限于 js。

2. 如果有想新加 repo 随时可以，不用担心破坏布局。

3. Contributions 指南的话，其实并没可维护的一个项目需要多数人参与，这个要看是否以后能有一个方向和落地。
4. 其他的你感兴趣都可以搞，没有什么太大的限制。

**Reply by qianxliu** [reply]
gitter是国际上git开发者常用的交流平台，类似qq和wechat，希望可以添加一下，正如我的第4个建议
我是做机器学习的，常见用gitter的
gitter类似Unity交流平台那个模式，非常简洁
gitter的消息反馈速度也很快

还有就是gitment实现主页评论功能，那样可以搜集足够多的数据进行进一步分析，这个很有趣

**Reply by xrkffgg** [reply]
😂😂 其实我现在平时工作都很忙的，QQ 微信都不用，一般是 钉钉 
gitter 这个你可以直接添加吗？还是一定要我添加

**Reply by qianxliu** [reply]
阿里巴巴吗？
![image](https://user-images.githubusercontent.com/53852392/121183126-531d2280-c896-11eb-8494-7bcd155e280e.png)
我没有 zoo-js org 的权限，只有这两个组织的

**Reply by qianxliu** [reply]
https://gitter.im/home/explore#createcommunity

**Reply by xrkffgg** [reply]
房间好像弄好了，然后呢？

**Reply by xrkffgg** [reply]
是阿里。。。

**Reply by qianxliu** [reply]
发链接就完了

**Comment by xrkffgg** [comment]
对了，还有一个，这个 https://github.com/zoo-js/zoo/issues/31 不要操作，保留着就行。
